installer/scripts/omsadmin.sh

```diff
@@ -352,6 +367,25 @@ set_proxy_setting()
     fi
 }
 
+set_omsagent_proc_limit()
```

Comment:
I am going to leave this PR for someone else to review who was initially involved with ulimit design discussions. I am still not clear what would be omsagent behavior if customer puts 1 as proc limit and how that can impact omiagent?